Dockerfile

```
@@ -1,6 +1,6 @@
-FROM python:3.8.5-buster
+FROM python:3.9.5-buster
```

```suggestion
FROM python:3.8.10-buster
```

can we stay on 3.8 for the time being? some ds libraries still don't work with 3.9 (like streamlit)

---

>🐍 Streamlit can now be installed on Python 3.9. Streamlit components are not yet compatible with Python 3.9 and must use version 3.8 or earlier.

I'll leave it at 3.8 for now though.